releases/latest/mysql-k8s-bundle.yaml

```diff
@@ -1,20 +1,34 @@
+bundle: kubernetes
+name: mysql-k8s-bundle
+description: MySQL operator bundle
+source: https://github.com/canonical/mysql-k8s-bundle
+issues: https://github.com/canonical/mysql-k8s-bundle/issues
 applications:
   mysql-k8s:
     channel: latest/edge
     charm: mysql-k8s
     constraints: arch=amd64
-    revision: 17
-    scale: 1
+    revision: 19
+    scale: 3
   mysql-router-k8s:
     channel: latest/edge
     charm: mysql-router-k8s
     constraints: arch=amd64
-    revision: 2
+    revision: 5
     scale: 1
     trust: true
-bundle: kubernetes
-name: mysql-k8s-bundle
+  tls-certificates-operator:
+    channel: latest/edge
+    charm: tls-certificates-operator
+    constraints: arch=amd64
+    revision: 11
+    scale: 1
+    options:
+      ca-common-name: canonical
+      generate-self-signed-certificates: true
```

Comment:
might be worth mentioning somewhere that in the bundle you default to self signed certs when using TLS, (see suggestion in README)

Reply:
suggestion commited